doc/Adhoc_Testing.md

```diff
@@ -0,0 +1,131 @@
+# Adhoc Testing
+
+The `bin/gen_large_datasets` utility can be utilies to generate synthesised:
+
+  * "collected" results for simulated VMWare and/or Libvirt hypervisor
+    topologiies.
+
+  * systems details including hostnames, system UUIDs, tokens and
+    created\_at time stamps for the relevant nodes in the simulated
```

Comment:
nit: created_at

Reply:
The backslash in  `created\_at` is to ensure that the `_` doesn't get treated as a Markdown tag start for _italics_.

Reply:
See https://github.com/rtamalin/scc-hypervisor-collector/blob/generate_large_data_sets/doc/Adhoc_Testing.md - the `_` is not rendered in the output.

In the past I have run into issues where an `_` link that in the text of a markdown block, if matched by another such `_` has lead to a block of text being rendered in italics.

Reply:
ah ok